src/main/java/com/databasepreservation/modules/siard/out/metadata/SIARD1MetadataExportStrategy.java

```diff
@@ -0,0 +1,786 @@
+
+	private static final String ENCODING = "UTF-8";
+	private static final String METADATA_XSD_RESOURCE_PATH = "/schema/siard1.xsd";
+
+	private final ContentPathExportStrategy contentPathStrategy;
+	private final MetadataPathStrategy metadataPathStrategy;
+	private final WriteStrategy writeStrategy;
+
+	public SIARD1MetadataExportStrategy(MetadataPathStrategy metadataPathStrategy,
+										ContentPathExportStrategy paths, WriteStrategy writer){
+		this.contentPathStrategy = paths;
+		this.writeStrategy = writer;
+		this.metadataPathStrategy = metadataPathStrategy;
+	}
+
+	@Override
+	public void writeMetadataXML(DatabaseStructure dbStructure, SIARDArchiveContainer container) throws ModuleException {
+
+		JAXBContext context;
+		try {
+			context = JAXBContext.newInstance("com.databasepreservation.modules.siard.common.jaxb.siard1");
+		} catch (JAXBException e) {
+			throw new ModuleException("Error loading JAXBContext", e);
+		}
+
+		SchemaFactory schemaFactory = SchemaFactory.newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI);
+		Schema xsdSchema = null;
+		try {
+			xsdSchema = schemaFactory.newSchema(Paths.get(getClass().getResource(METADATA_XSD_RESOURCE_PATH).getPath()).toFile());
+		} catch (SAXException e) {
+			throw new ModuleException("XSD file has errors: " + getClass().getResource(METADATA_XSD_RESOURCE_PATH).getPath(), e);
+		}
+
+		SiardArchive xmlroot = jaxbSiardArchive(dbStructure);
+		Marshaller m;
+		try {
+			m = context.createMarshaller();
+			m.setProperty(Marshaller.JAXB_FORMATTED_OUTPUT, Boolean.TRUE);
+		    m.setProperty(Marshaller.JAXB_ENCODING, ENCODING);
+			m.setProperty(Marshaller.JAXB_SCHEMA_LOCATION, "http://www.bar.admin.ch/xmlns/siard/1.0/metadata.xsd metadata.xsd");
+
+			//m.marshal(xmlroot, System.out);
+
+			m.setSchema(xsdSchema);
+			OutputStream writer = writeStrategy.createOutputStream(container, metadataPathStrategy.getMetadataXmlFilePath());
+			m.marshal(xmlroot, writer);
+			writer.close();
+		} catch (JAXBException e) {
+			throw new ModuleException("Error while Marshalling JAXB", e);
+		} catch (IOException e) {
+			throw new ModuleException("Error while closing the data writer", e);
+		}
+	}
+
+	@Override
+	public void writeMetadataXSD(DatabaseStructure dbStructure, SIARDArchiveContainer container) throws ModuleException {
+		// prepare to write
+		OutputStream out = writeStrategy.createOutputStream(container, metadataPathStrategy.getMetadataXsdFilePath());
+		BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(out));
+
+		// prepare to read
+		Path xsdSchema = Paths.get(getClass().getResource(METADATA_XSD_RESOURCE_PATH).getPath());
+		Reader reader = null;
+		try {
+			 reader = Files.newBufferedReader(xsdSchema, StandardCharsets.UTF_8);
+		} catch (IOException e) {
+			throw new ModuleException("Could not read from " + getClass().getResource(METADATA_XSD_RESOURCE_PATH).getPath(), e);
+		}
+
+		// read everything from reader into writer
+		try {
+			IOUtils.copy(reader, writer);
```

Comment:
Change this to use Files.copy(), reading directly from xsdSchema path: http://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#copy(java.nio.file.Path,%20java.io.OutputStream)